app/views/carts/_empty_cart.html.haml

```diff
@@ -0,0 +1,4 @@
+.ui.compact.raised.segment{:style => "margin: auto; top: 25%; width: 20%; padding: 10px"}
```

Comment:
you see, these inline styles look really bad. Separate file
also what's the problem with `nt{:style =>` syntax?
God gave you `{ style: 'mar...` syntax, use it